AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TaskController.cs

```diff
@@ -55,6 +58,15 @@ public IActionResult Create(int activityId, TaskEditModel model)
                 ModelState.AddModelError("EndDateTime", "Ending time cannot be earlier than the starting time");
             }
 
+            var activity = _dataAccess.GetActivity(activityId);
+            if ((model.StartDateTime < activity.StartDateTime || model.EndDateTime > activity.EndDateTime) && model.IgnoreWarning == false)
+            {
+                ModelState.AddModelError("", "Although valid, task date and time is out the range of activity date and time of " + 
+                    activity.StartDateTime.LocalDateTime.ToString() + " to " + activity.EndDateTime.LocalDateTime.ToString());

```

Comment:
When hosting in Azure, the server time zone is UTC so this will actually display in UTC time instead of the user's local time. What we actually want is to display the time in the timezone of the campaign. In this case activity.StartDateTime.ToString() would be sufficient.